lib/cli/commands/tree.js

```diff
@@ -90,12 +90,13 @@ tree.handler = async function(argv) {
 				if (project.isFrameworkProject()) {
 					name = chalk.blue(name);
 				}
-				console.log(
+				process.stderr.write(
```

Comment:
Since the `ui5 tree` output generated here is the sole result of the operation (one might call it "payload"), it must be written to **stdout**. This allows for piping operations, for example to create a file using `ui5 tree > output.txt`.

This way the file won't contain any of the other log messages since those are written to stderr.

I think this affects everything from this comment to the next one (until `argv.perf`)

Reply:
Done